src/components/presentational/SegmentedControl/SegmentedControl.css

```diff
@@ -1,7 +1,6 @@
 .mdhui-segmented-control {
     border-radius: 32px;
     border: solid 1px var(--mdhui-color-primary);
-    background: var(--mdhui-background-color-0);
```

Comment:
Curious about this removal.  I think it looks a little better with the background color slightly different than the parent's background color.  Was there a contrast concern here?